README.md

```diff
@@ -151,11 +151,24 @@ This test suite expects following environment variables set:
  - `STORAGE_API_TOKEN_ADMIN_3_IN_OTHER_ORGANIZATION` - Storage API token associated to other admin as `STORAGE_API_TOKEN` and project in the other organization as `STORAGE_API_TOKEN`.
 You can export variables manually or you can create and fill file `set-env.mixed.sh` as copy of attached `set-env.mixed.template.sh`.
 
-Than  you can run tests:
+Then you can run tests:
 
 `source ./set-env.mixed.sh && docker-compose run --rm dev vendor/bin/phpunit --testsuite backend-mixed'`
 
+## Running test from PHPStorm
 
+The whole test suite is quite big and it can take few hours. So it is a good idea to run just a testcase which you are interested in from PHPStorm, or you can run them from a console (using `--filter` option). 
+
+**How to set up PHPStorm for running tests:**
+- go to Settings / Languages & Frameworks / PHP
+- row CLI -> three dots
+- Plus button -> `From Docker, Vagrant, VM, WSL, Remote...`
+- Select `Docker Compose`; Service `dev-xdebug`; Environment variables define value from your `set-env.php`
+    - an easy way how to do it is copy content of `set-env.php` without `export ` prefix -> click on the icon in  Env. vars. -> click on the paste icon. It should pass all the key=value entries in the window. If it doesn't work, set them manually.
+- Set Path mappings to `<Project root> -> /code` (`<Project root>` is an absolute path to the project directory) 
+- hint: create different interpreters for different environments
+
+_Note: see [this link](https://blog.jetbrains.com/phpstorm/2017/07/new-docker-compose-support-in-phpstorm-2017-2/) for more information and screenshots about the description above._
```

Comment:
Tohle mi přijde divný. Ale klidně to nech, jestli myslíš, že to pomůže. 

Já bych čekal spíš odkaz na https://www.jetbrains.com/help/phpstorm/configuring-remote-interpreters.html nebo https://confluence.jetbrains.com/pages/viewpage.action?pageId=57294700#WorkingwithRemotePHPInterpretersinPhpStorm-PHPUnitandRemotePHPInterpreters nebo https://www.jetbrains.com/help/phpstorm/using-phpunit-framework.html#configure-phpunit-manually